templates/distribution/manifests/auth/secrets/dex.yml.tpl

```diff
@@ -2,14 +2,14 @@
   {{ if .spec.distribution.modules.auth.overrides.ingresses.dex.host -}}
     {{ print "https://" .spec.distribution.modules.auth.overrides.ingresses.dex.host }}
   {{- else -}}
-    {{ print "https://login." .spec.distribution.modules.ingress.baseDomain }}
+    {{ print "https://login." .spec.distribution.modules.auth.baseDomain }}
   {{- end }}
 {{- end -}}
 {{- define "pomeriumHost" }}
   {{- if .spec.distribution.modules.auth.overrides.ingresses.pomerium.host -}}
     {{ print "https://" .spec.distribution.modules.auth.overrides.ingresses.pomerium.host "/oauth2/callback" }}
   {{- else -}}
-    {{ print "https://pomerium.internal." .spec.distribution.modules.ingress.baseDomain "/oauth2/callback" }}
+    {{ print "https://pomerium." .spec.distribution.modules.auth.baseDomain "/oauth2/callback" }}
```

Comment:
is `auth.baseDomain` correct? I don't see it being added to the schema

Reply:
Yes it's correct. I don't know how to add it to the schema 😄

maybe @Al-Pragliola can help us after the merge